qiskit_dynamics/solve.py

```diff
@@ -201,320 +228,138 @@ def solve_lmde(
     Results are returned as a :class:`OdeResult` object.
 
     Args:
-        generator: Representaiton of generator function :math:`G(t)`.
+        generator: Representation of generator function :math:`G(t)`.
```

Comment:
I want to avoid automatically doing that, as working with the vectorized Lindblad equation is potentially expensive. Also, `solve_lmde` and `solve_ode` no longer make these kinds of checks. I can at least modify this to say that it must be in standard form if a subclass of `BaseGeneratorModel`.